azkaban-common/src/main/java/azkaban/imagemgmt/models/HPFlowOwnership.java

```diff
@@ -0,0 +1,17 @@
+package azkaban.imagemgmt.models;
+
+/**
+ * This class represents HP Flow managers' metadata
+ */
+public class HPFlowOwnership extends BaseModel {
+  // HP flow management owner name
+  private String name;
```

Comment:
Please rename this variable to owner or username.

Reply:
Will do.